tex/gregoriotex.sty

```diff
@@ -123,14 +123,13 @@
   \gre@changestyle{normalstafflines}{\color[named]{#1}}[\relax]%
 }
 
-
 \def\grecoloredlines#1#2#3{%
-  \gre@warning{\protect\grecoloredlines\space is deprecated.\MessageBreak Use \protect\gresetlinecolor\space instead.}%
+  \gre@deprecated{\protect\grecoloredlines}{\protect\gresetlinecolor}%
   \gre@changestyle{normalstafflines}{\color[RGB]{#1,#2,#3}}%
 }
 
 \def\greredlines{%
-  \gre@warning{\protect\greredlines\space is deprecated.\MessageBreak Use \protect\gresetlinecolor{gregoriocolor}\space instead.}%
+  \gre@deprecated{\protect\greredlines}{\protect\gresetlinecolor{gregoriocolor}%
```

Comment:
Sorry for being piecemeal, but I didn't notice it before.  The braces are unbalanced in #2.  Same below.